Makefile

```diff
@@ -210,6 +210,7 @@ hooks: # Install Git hooks
 	cp hack/pre-commit.sh .git/hooks/pre-commit
 
 .PHONY: test
+unexport PROTOKUBE_IMAGE NODEUP_URL KOPS_BASE_URL DNSCONTROLLER_IMAGE KOPS_RUN_OBSOLETE_VERSION ENV_VAR_CNI_VERSION_URL AWS_REGION AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY AWS_SESSION_TOKEN AWS_PROFILE
```

Comment:
What's the scope of unexport here? Will it only apply to the `test` recipe, or every recipe below it as well?

Reply:
@rifelpet it appears that unexport has global scope, so we should put it at the top of the file for clarity.